src/filters/number.rs

```diff
@@ -62,26 +62,43 @@ pub fn filesizeformat(value: Value, _: HashMap<String, Value>) -> Result<Value>
 }
 
 
-/// Returns a formatted timestamp according to the given `format` argument.
-/// `format` defaults to the ISO 8601 `YYYY-MM-DD` format
+/// Returns a formatted time according to the given `format` argument.
+/// `format` defaults to the ISO 8601 `YYYY-MM-DD` format.
+///
+/// Input can be an i64 timestamp (seconds since epoch) or an RFC3339 string
+/// (default serialization format for chrono::DateTime).
+///
 /// Time formatting syntax is inspired from strftime and a full reference is available
 /// on [chrono docs](https://lifthrasiir.github.io/rust-chrono/chrono/format/strftime/index.html)
 pub fn date(value: Value, args: HashMap<String, Value>) -> Result<Value> {
```

Comment:
since that now works on both number and string, can you put it in `common.rs` and use pattern matching on the value like in that file?